src/resources/postcss/tribe-common-admin/tec_baseline_admin/_main.pcss

```diff
@@ -0,0 +1,20 @@
+#tec-admin-page {
```

Comment:
Instead of applying all styles directly to `#tec-admin-page`, would it make sense to introduce a class for layout variations? For example, we could use `#tec-admin-page .default_two_column` for the current layout. By adding or changing the class, we could define alternate layouts, such as `#tec-admin-page .help_hub`, with styles tailored specifically to those pages.

This could prevent conflicts when working on new layouts, similar to what I experienced with Help Hub.

Reply:
The ID is there to prevent bleed while we work out the kinks.
Yes, classes for layout variations makes good sense.